README.md

```diff
@@ -39,6 +39,20 @@ chanjo calculate mean
 {"metrics": {"completeness_10": 90.92, "mean_coverage": 193.85}, "sample_id": "sample1"}
 ```
 
+### Docker
+
+When running the dockerized version of [Chanjo](https://hub.docker.com/r/clinicalgenomics/chanjo) the setup process is slightly different. Chanjo depends on a configuration file `config.yaml` and sqlite database `chanjo.coverage.sqlite3` which are created at initialization. Docker [volumes](https://docs.docker.com/storage/volumes/) can be used to make these files persistent and backupable. The following examples demonstrate how to setup Chanjo using the default definition of exons. The config file and database are stored on the host in a folder called config. Other exon definitions can be used by mounting them to the container. 
```

Comment:
It's not only sqlite that you can use to initiate the demo database, for instance we use MariDB in the docker-compose file. Perhaps include also this?